app/main.js

```diff
@@ -47,7 +47,9 @@ function forceShowWindow() {
 
 function createWindow() {
   // Create the browser window.
-  win = new BrowserWindow({width: 500, height: 350});
+  var w = 1500;
```

Comment:
Was this intentional?  I think a little bigger is OK, but 1500x1000 seems way too big by default to me.